### DISCUSSION TITLE
featured_image on content pages

### DISCUSSION THREAD

**Discussion by maroliict** [binokochumolvarghese/lightbi-hugo#68] [Ideas]
Hi,
Thanks for your awesome template. It works really well!

I have a question/need. I want to add a Hero Image (or Featured_image) per post/page. What is the best way to do this? I must admit that I can read some HTML but I am still strugling with how the LightBi is setup. A litle help and sample code would be awesome.

btw: Might be that it is implemented, but with a different name.. .then please point me in the right direction

**Comment by binokochumolvarghese** [comment]
Are you mentioning the default image in the post like [this](https://lightbi-hugo-theme.netlify.app/post/2020/jan/build-hugo-blog/). If it is not please let me know.

**Reply by maroliict** [reply]
no.. it is more like this the hero header on [this page](https://rob-litjens.nl) .. The image changes on every page. It is defined in the front-matter like featured_image...

**Reply by binokochumolvarghese** [reply]
I'm sorry, as of now there is no option to do that.
